shared/src/operatingPhase.ts

```diff
@@ -24,21 +26,26 @@ export const OperatingPhases: OperatingPhase[] = [
   {
     id: "GUEST_MODE",
     hasCompletedTaxRegistration: false,
+    displayCertifications: false,
   },
   {
     id: "FORMED_AND_REGISTERED",
     hasCompletedTaxRegistration: true,
+    displayCertifications: true,
   },
   {
     id: "NEEDS_TO_FORM",
     hasCompletedTaxRegistration: false,
+    displayCertifications: false,
   },
   {
     id: "NEEDS_TO_REGISTER_FOR_TAXES",
     hasCompletedTaxRegistration: false,
+    displayCertifications: false,
   },
   {
     id: "UP_AND_RUNNING",
     hasCompletedTaxRegistration: true,
+    displayCertifications: true,
```

Comment:
why should we track this separately from `hasCompletedTaxRegistration` if they are always the same?

Reply:
If we want to keep them separate, we should rename 'hasCompletedTaxRegistration' to 'dispalyTaxProfileFields'
My thinking was to have flags for specific fields/control to make it easier to change/separate if we need to